data_diff/database.py

```diff
@@ -714,6 +716,65 @@ def normalize_value_by_type(self, value: str, coltype: ColType) -> str:
 HELP_PRESTO_URI_FORMAT = "presto://<user>@<host>/<catalog>/<schema>"
 
 
+@dataclass
+class MatchUriPath:
```

Comment:
I think a couple of quick unit tests for this class would be helpful

Reply:
Done